valhalla/baldr/graphreader.h

```diff
@@ -637,16 +637,19 @@ class GraphReader {
   /**
    * Method to get the begin node of an edge by using its opposing edges end node
    * @param edge    the edge whose begin node you want
-   * @param tile    reference to a pointer to a const tile
+   * @param tile    reference to a pointer to a const tile containing the begin node
    * @return        returns GraphId of begin node of the edge (empty if couldn't find)
    */
   GraphId GetBeginNodeId(const DirectedEdge* edge, graph_tile_ptr& tile) {
```

Comment:
If this isn't actually going to mutate the tile pointer any mroe, the signature should be changed `const graph_tile_ptr&`

Reply:
Oh I see, I missed the added call to `GetGraphTile(opp_edge->endnode(), tile);`.